src/Lykke.Job.CandlesHistoryWriter.Services/HistoryMigration/TradesMigrationManager.cs

```diff
@@ -62,60 +63,75 @@ public bool Migrate(ITradesMigrationRequest request)
+                                    ? new TradesCandleBatch(migrationItem.AssetId, interval, tradesBatch)
+                                    : new TradesCandleBatch(migrationItem.AssetId, interval, smallerCandles);
+
+                                ExtendStoredCandles(ref currentCandles);
+
+                                Health.AssetReportItems[migrationItem.AssetId].SummarySavedCandles +=
+                                    currentCandles.CandlesCount;
+
+                                // We can not derive month candles from weeks for they may lay out of the borders
+                                // of the month. While generating a month candles, we should use day candles as a
+                                // data source instead.
+                                if (interval != CandleTimeInterval.Week)
+                                    smallerCandles = currentCandles;
+
+                                // Preffered to perform inserting in parralel style.
+                                batchInsertTasks.Add(
+                                    _candlesHistoryRepository.InsertOrMergeAsync(
+                                        currentCandles.Candles.Values,
+                                        currentCandles.AssetId,
+                                        TradesCandleBatch.PriceType,
+                                        currentCandles.TimeInterval));
+                            }
+
+                            Health.AssetReportItems[migrationItem.AssetId].SummaryFetchedTrades += batchCount;
+
+                            Task.WaitAll(batchInsertTasks.ToArray());
+
+                            await _log.WriteInfoAsync(nameof(TradesMigrationManager), nameof(Migrate),
+                                $"Batch of {batchCount} records for {migrationItem.AssetId} processed.");
                         }
 
-                        Health.AssetReportItems[migrationItem.AssetId].SummaryFetchedTrades += batchCount;
-
-                        Task.WaitAll(batchInsertTasks.ToArray());
+                        Health.State = TradesMigrationState.Finished;
 
                         await _log.WriteInfoAsync(nameof(TradesMigrationManager), nameof(Migrate),
-                            $"Batch of {batchCount} records for {migrationItem.AssetId} processed.");
+                            $"Migration for {migrationItem.AssetId} finished. Total records migrated: {Health.AssetReportItems[migrationItem.AssetId].SummaryFetchedTrades}, total candles stored: {Health.AssetReportItems[migrationItem.AssetId].SummarySavedCandles}.");
+                    }
+                    catch (Exception ex)
+                    {
+                        Health.State = TradesMigrationState.Error;
+                        await _log.WriteErrorAsync(nameof(TradesSqlHistoryRepository), nameof(DoMigrateAsync), ex);
+                        await _log.WriteInfoAsync(nameof(TradesMigrationManager), nameof(Migrate),
```

Comment:
This log message is excess

Reply:
The matter is, that's a "closing parenthes" message. Previously we declared the start of the process in Info, and now should (?) declare the ending. Are you sure we do not need to?